static/coffee/flows/controllers.coffee

```diff
@@ -1805,6 +1843,27 @@ NodeEditorController = ($rootScope, $scope, $modalInstance, $timeout, $log, Flow
     if $scope.action.webhook_headers.length == 0
       $scope.addNewActionWebhookHeader()
 
+  $scope.addNewQuickReply = ->
+    if $scope.actions_quick_reply.length < 11
+      $scope.container_operation_visible = false
+      if Object.keys($scope.action.quick_replies).length < 1
+        $scope.actions_quick_reply.push({title:''})
+      else
+        for lang of $scope.action.quick_replies
+          $scope.action.quick_replies[lang].push({title:''})
```

Comment:
Do we need to push empty translations here? Seems better to only add actual translations to the flow definition